droidlet/interpreter/robot/tasks.py

```diff
@@ -359,6 +365,159 @@ def step(self):
     def __repr__(self):
         return "<TrajectorySaverTask {}>".format(self.target)
 
+class CuriousExplore(TrajectorySaverTask):
+    """use slam to explore environemt, but also examine detections"""
+
+    def __init__(self, agent, task_data):
+        super().__init__(agent, task_data)
+        self.steps = ["not_started"] * 2
+        self.task_data = task_data
+        self.goal = task_data.get("goal", (19,19,0))
```

Comment:
why (19,19, 0) ?

Reply:
this was just the previous default, but getting rid of this so that we always have to ensure setting the goal in default_behaviors.py